app/views/additional_payments/claims/induction_completed.html.erb

```diff
@@ -1,42 +1,38 @@
-<% content_for(:page_title, page_title(t("additional_payments.questions.induction_completed.heading"), journey: current_journey_routing_name, show_error: current_claim.errors.any?)) %>
-<% path_for_form = current_claim.persisted? ? claim_path(current_journey_routing_name) : claims_path(current_journey_routing_name) %>
+<% content_for(:page_title, page_title(t("additional_payments.questions.induction_completed.heading"), journey: current_journey_routing_name, show_error: @form.errors.any?)) %>
+<% path_for_form = @form.persisted? ? claim_path(current_journey_routing_name) : claims_path(current_journey_routing_name) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: { "eligibility.induction_completed": "claim_eligibility_attributes_induction_completed_true" }) if current_claim.errors.any? %>
-    <%= form_for current_claim, url: path_for_form do |form| %>
-      <%= form_group_tag current_claim do %>
-        <%= form.fields_for :eligibility, include_id: false do |fields| %>
+    <%= render("shared/error_summary", instance: @form, errored_field_id_overrides: { "induction_completed": "claim_induction_completed-error" }) if @form.errors.any? %>
+    <%= form_for @form, url: path_for_form do |f| %>
```

Comment:
The forms will always be `persited? #=> true` so we can lose the `path_for_form` variable and remove the line above where we set it
```suggestion
    <%= form_for @form, url: claim_path(current_journey_routing_name) do |f| %>
```